tests/ci-files/ci-kind-config.yaml

```diff
@@ -0,0 +1,7 @@
+kind: Cluster
```

Comment:
is this going outside of pipleines?

Reply:
It's so we are using the same configuration in github and gitlab